index.html

```diff
@@ -217,5 +217,19 @@ <h2>
             </div>
         </section>
         </main>
+
+        <footer> 
+            <section class="contact">
+                <h2>Contact me</h2>
+              <p>I'm always interested in hearing about new projects,
+              so if you'd like to chat please get in touch.</p>
+              <form action="https://formspree.io/f/{form_id}" method="post">
+                <input type="text" name="fullname" placeholder="Yeremias" maxlength="30" required>
+                <input type="email" name="email" placeholder="natajayanj14@gmail.com" required>
+                <textarea id="message" maxlength="500" placeholder="Write your message here" required></textarea>
+                <button type="submit" id="get-in-touch" >Get in touch</button>
```

Comment:
:red_circle:  Kindly make sure this button matches the Figma design. You can round your button's corners and also set your border to none

Your Design
![button](https://user-images.githubusercontent.com/26736582/199834371-975628e4-be72-4f77-9500-d18223443352.png)
Figma Design

![buttonfigma](https://user-images.githubusercontent.com/26736582/199834401-7803b5c7-f7d3-4e4b-be69-9a459b55afa2.png)

- [ ] :red_circle:  Kindly implement the button interactions (enable, hover, etc.).
![buuton](https://user-images.githubusercontent.com/26736582/199839104-93b1251f-0d75-4e4f-878a-8d3ffdfb70aa.png)

Reply:
- [] Please double-check because that is not my button.